st2actions/st2actions/cmd/st2resultstracker.py

```diff
@@ -0,0 +1,70 @@
+import eventlet
+import os
+import sys
+
+from oslo.config import cfg
+
+from st2common import log as logging
+from st2common.models.db import db_setup
+from st2common.models.db import db_teardown
+from st2common.constants.logging import DEFAULT_LOGGING_CONF_PATH
+from st2actions import config
+from st2actions import resultstracker
+
+
+LOG = logging.getLogger(__name__)
+
+
+eventlet.monkey_patch(
+    os=True,
+    select=True,
+    socket=True,
+    thread=False if '--use-debugger' in sys.argv else True,
```

Comment:
I don't think we really use this but if you care you user a debugger keep it around.